packages/nx-plugin/src/executors/autorun/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { globalConfig, persistConfig, uploadConfig } from '../internal/config';
+import { NormalizedExecutorContext } from '../internal/context';
+import {
+  AutorunCommandExecutorOnlyOptions,
+  AutorunCommandExecutorOptions,
+} from './schema';
+
+export function parseAutorunExecutorOnlyOptions(
+  options: Partial<AutorunCommandExecutorOnlyOptions>,
+): AutorunCommandExecutorOnlyOptions {
+  const { projectPrefix, dryRun, onlyPlugins } = options;
+  return {
+    ...(projectPrefix && { projectPrefix }),
+    ...(dryRun != null && { dryRun: dryRun }),
```

Comment:
```suggestion
    ...(dryRun != null && { dryRun }),
```

:top: I'm sure I've seen a lint rule for that, but can't remember what it was called :sweat_smile:

Reply:
One of this?

-  "prefer-object-spread/prefer-object-spread": "error",
- "object-shorthand": ["error", "always"],
- "no-useless-rename": "error"

Reply:
Yes, that's what I was looking for :heart: It's even in the core set of rules, don't know how I missed it :sweat_smile: Created an issue to add those rules: https://github.com/code-pushup/eslint-config/issues/19